aibolit/__main__.py

```diff
@@ -120,10 +120,24 @@ def train():
         required=False,
         default=False
     )
+    parser.add_argument(
+        "--target-metric",
+        choices=["rfc", "fanout", "diameter", "cognitive"],
+        default="cognitive",
+        help="Select target metric. Available metrics: rfc, fanout, diameter, cognitive. "
+             "Default is cognitive."
+    )
     args = parser.parse_args(sys.argv[2:])
     if not args.skip_collect_dataset:
         collect_dataset(args)
-    train_process()
+
+    metric_name_to_code = {
+        "rfc": "M9",
+        "fanout": "M10",
+        "diameter": "M6",
+        "cognitive": "M4",
+    }
```

Comment:
If possible, I'd add this data to aibolit/config.py

Reply:
It is possible, but this dict is specific to command line arguments of train, so I would keep it close to that method. Anyway it use different names than in config.